Index: AndroidManifest.xml
===================================================================
--- AndroidManifest.xml	(revision 92)
+++ AndroidManifest.xml	(working copy)
@@ -8,7 +8,9 @@
         android:minSdkVersion="11"
         android:targetSdkVersion="17" />
 	<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
-    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.READ_CONTACTS"/>
+    <uses-permission android:name="android.permission.SEND_SMS"/>
 
     <application
         android:allowBackup="true"
Index: src/com/calerem/API/c_sms_API.java
===================================================================
--- src/com/calerem/API/c_sms_API.java	(revision 0)
+++ src/com/calerem/API/c_sms_API.java	(working copy)
@@ -0,0 +1,32 @@
+/**
+ * 
+ */
+package com.calerem.API;
+import android.telephony.SmsManager;
+
+/**
+ * Connects to android SMS API.
+ * @author DarkParadise
+ * @version 1.0
+ */
+public class c_sms_API {
+	/**
+	 * Sends a silent SMS.
+	 * @param v_phone Recipient Phone
+	 * @param v_text SMS Message Text Maximum 160 Characters
+	 * @return boolean on success
+	 */
+	public boolean f_SendSMS(String v_phone,String v_text)
+	{
+		if(v_text.length() <= 160)
+		{
+			SmsManager sms = SmsManager.getDefault();
+			sms.sendTextMessage(v_phone, null, v_text, null, null);
+			return true;	
+		}
+		else
+		{
+			return false;
+		}
+	}
+}
